internal/plugintest/working_dir.go

```diff
@@ -151,6 +157,80 @@ func (wd *WorkingDir) SetConfig(ctx context.Context, cfg teststep.Config, vars c
 	return nil
 }
 
+// SetQuery sets a new query configuration for the working directory.
+//
+// This must be called at least once before any call to Init or Query Destroy
+// to establish the query configuration. Any previously-set configuration is
+// discarded and any saved plan is cleared.
+func (wd *WorkingDir) SetQuery(ctx context.Context, cfg teststep.Config, vars config.Variables) error {
+	// Remove old config and variables files first
+	d, err := os.Open(wd.baseDir)
+
+	if err != nil {
+		return err
+	}
+
+	defer d.Close()
+
+	fi, err := d.Readdir(-1)
+
+	if err != nil {
+		return err
+	}
+
+	for _, file := range fi {
+		if file.Mode().IsRegular() {
+			if filepath.Ext(file.Name()) == ".warioform" || filepath.Ext(file.Name()) == ".json" || filepath.Ext(file.Name()) == ".hcl" {
```

Comment:
Do you think it makes sense to be more specific with the file extension here as well, also
>.warioform

😬 

```suggestion
			if filepath.Ext(file.Name()) == ".warioform" || filepath.Ext(file.Name()) == ".json" || filepath.Ext(file.Name()) == ".tfquery.hcl" {
```